vpr/src/route/connection_router.h

```diff
@@ -46,8 +45,13 @@ class ConnectionRouter : public ConnectionRouterInterface {
         , router_stats_(nullptr)
         , router_debug_(false) {
         heap_.init_heap(grid);
-        heap_.set_prune_limit(rr_nodes_.size(), kHeapPruneFactor * rr_nodes_.size());
         only_opin_inter_layer = (grid.get_num_layers() > 1) && inter_layer_connections_limited_to_opin(*rr_graph);
+        rr_node_R_upstream.resize(rr_node_route_inf.size());
+        rcv_path_data.resize(rr_node_route_inf.size());
+    }
+
+    ~ConnectionRouter() {
+        VTR_LOG("Serial Connection Router is being destroyed. Time spent computing SSSP: %.3f seconds.\n", this->sssp_total_time.count() / 1000000.0);
```

Comment:
To make the conversion (to seconds) more obvious, I chose to use the conversion method provided by `std::chrono` instead of dividing by USEC_PER_SEC, since the timer itself comes from `std::chrono`.
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/2966d66b1e141c9b2c771557fcafe0ca285a20ab/vpr/src/route/connection_router.h#L52-L55